packages/monocdk/package.json

```diff
@@ -269,6 +270,7 @@
     "@types/fs-extra": "^8.1.1",
     "@types/node": "^10.17.47",
     "cdk-build-tools": "0.0.0",
+    "cfn-include-mapping-rewriter": "0.0.0",
```

Comment:
Keep the name terse. just call it `cfninclude-rewriter`.

Reply:
Done.